RUFAS/biophysical/manure/storage/slurry_storage_outdoor.py

```diff
@@ -0,0 +1,144 @@
+from copy import copy
+
+from math import inf
+
+from RUFAS.biophysical.manure.storage.storage import (
+    Storage,
+    DEFAULT_PH_FOR_AMMONIA,
+    STORAGE_COVER_NITROUS_OXIDE_EMISSIONS_FACTOR_MAPPING,
+)
+from RUFAS.biophysical.manure.storage.storage_cover import StorageCover
+from RUFAS.current_day_conditions import CurrentDayConditions
+from RUFAS.data_structures.animal_to_manure_connection import ManureStream
+from RUFAS.general_constants import GeneralConstants
+from RUFAS.time import Time
+from RUFAS.units import MeasurementUnits
+
+SLURRY_MANURE_DENSITY = 990
+"""The density of slurry manure (kg/:math:`m^3`)."""
+
+STORAGE_HSC = 4.1
+"""Housing specific constant for manure storage (s/m)."""
+
+METHANE_TO_METHANE_CARBON_DIOXIDE_RATIO: float = 9.25
+"""
+The mass conversion factor from methane to methane and carbon dioxide emitted from stored manure, based on a molar
+ratio of 1:3 (methane : carbon dioxide).
+"""
+
+
+class SlurryStorageOutdoor(Storage):
+    def __init__(
+        self,
+        name: str,
+        is_housing_emissions_calculator: bool,
+        cover: StorageCover,
+        storage_time_period: int | None,
+        surface_area: float,
+        nitrous_oxide_emissions_factor: float,
+        capacity: float = inf,
+    ):
+        super().__init__(
+            name=name,
+            is_housing_emissions_calculator=is_housing_emissions_calculator,
```

Comment:
```suggestion
            is_housing_emissions_calculator=False
```
I think this should be false for everything except Handlers and I'm not sure if we need to have it be an init param - I think we could just pass in False and not worry about it.

Reply:
@elle-andreen please confirm this.

Reply:
From the design doc:

![Screenshot 2025-03-21 at 11 05 44 AM](https://github.com/user-attachments/assets/b5404dbb-a3b8-442f-a889-8cf834f5a0ca)